installer/netdata-uninstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#shellcheck disable=SC2181
+
+# this script will uninstall netdata
+
+# Variables needed by script:
+#  - NETDATA_PREFIX
+#  - NETDATA_ADDED_TO_GROUPS
+
+if [ "$1" != "--force" ]; then
+	echo >&2 "This script will REMOVE netdata from your system."
+	echo >&2 "Run it again with --force to do it."
+	exit 1
+fi
+
+#shellcheck source=/dev/null
+source installer/.environment.sh || exit 1
+#shellcheck source=/dev/null
+source installer/functions.sh || exit 1
```

Comment:
It would be safer to include them in the opposite order.
just in case `functions.sh` uses a variable used by `.environment.sh`.

Reply:
only updater and uninstaller are allowed to use those variables.